src/components/Modals/UseItemOutCombat.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import Modal from 'react-native-modal';
+import { CustomButton } from '../CustomButton';
+import { TCaughtLoomieToRender, TCaughtLoomies, TItem } from '@src/types/types';
+import { EmptyMessage } from '../EmptyMessage';
+import { LoomiesGrid } from '../CaughtLoomiesGrid/LoomiesGrid';
+import { navigate } from '@src/navigation/RootNavigation';
+import { useToastAlert } from '@src/hooks/useToastAlert';
+import { useItemOutCombat } from '@src/services/items.services';
+import { getLoomiesRequest } from '@src/services/loomies.services';
+
+interface IProps {
+  selectedItem: TItem;
+  isVisible: boolean;
+  toggleVisibilityCallback: () => void;
+  closeModalItem: () => void;
+}
+
+export const UseItemOutCombatModal = ({
+  selectedItem,
+  isVisible,
+  toggleVisibilityCallback,
+  closeModalItem
+}: IProps) => {
+  const { showErrorToast, showSuccessToast } = useToastAlert();
+  const [team, setTeam] = useState<string>();
```

Comment:
What is this state. Seems it isn't a team, you are assigned a Loomie Id as the value.

Reply:
I forgot to change the name, it's to select a loomie to which to apply the item